src/main/kotlin/io/github/enyason/predictable/PredictableValidator.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.enyason.io.github.enyason.predictable
```

Comment:
Check the package

Reply:
how do you mean?

Reply:
This part *io.github.enyason* is repeated

Reply:
got it.